Competition2018/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -177,31 +176,34 @@ private double ticksToInches(double ticks) {
      */
 
     boolean isCloseToMaxmumHeight() {
-
-        if (getCurrentHeight() >= maxHeightInInches.get() * 0.9) {
+        if (getCurrentHeightInInches() >= maxHeightInInches.get() * 0.9) {
```

Comment:
Why `* 0.9`? "magic numbers" are generally to be avoided because they are confusing to a reader.

Reply:
done